src/utils/object.js

```diff
@@ -120,6 +120,8 @@ export function cleanUp(obj) {
             return cleaned;
           }
         }
+
+        return false;
```

Comment:
As opposed to an implicit `return undefined`?

Reply:
Commit dropped, since we updated to dashboard's eslint configs and this no longer shows up :)